deploy/kube/mixer.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: mixer
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: mixer
+      annotations:
+        prometheus_port: "42422"
```

Comment:
The prometheus.yaml included in this PR is using meta labels:

````
   kubernetes_sd_configs:
    - role: pod
   relabel_configs:
    # we read the pod annotations "prometheus_port" from kubernetes, and concat it with the address
    - source_labels: [__address__, __meta_kubernetes_pod_annotation_prometheus_port]
      action: replace
      regex: ([^:]+)(?::\d+)?;(\d+)
      replacement: $1:$2
      target_label: __address__
````

I can swap over to the `service` role and rework the relabeling.